TangemSdk/TangemSdk/Common/NFC/NFCReader.swift

```diff
@@ -247,6 +260,11 @@ extension NFCReader: CardReader {
     }
 
     func stopSession(with errorMessage: String? = nil) {
+        guard !isBeingStopped || (readerSession?.isReady ?? false) else {
+            return
+        }
```

Comment:
не критично, но думаю проще читается такой вариант:
```suggestion
        if isBeingStopped {
            return
        }
        
        guard (readerSession?.isReady ?? false) else {
            return
        }
```

Reply:
слегка переделал